core/imt/src/TThreadExecutor.cxx

```diff
@@ -92,7 +92,7 @@ namespace ROOT {
    {
       if (nThreads == 0)
          Error("TThreadExecutor", "Number of threads should be positive");
-      if (ROOT::Internal::TPoolManager::GetPoolSize() != nThreads) {
+      if (ROOT::Internal::TPoolManager::GetPoolSize() > 0 && ROOT::Internal::TPoolManager::GetPoolSize() != nThreads) {
```

Comment:
Please use `git rebase -i` and combine the two commits into one (or squash when merging).

Reply:
Will do